packages/liveblocks-react-comments/src/components/InboxNotification.tsx

```diff
@@ -360,10 +360,10 @@ const InboxNotificationThread = forwardRef<
     const {
       useRoomInfo,
       useInboxNotificationThread,
-      [kInternal]: { useCurrentUserId },
+      [kInternal]: { useCurrentUserIdFromClient },
     } = useLiveblocksContextBundle();
     const thread = useInboxNotificationThread(inboxNotification.id);
-    const currentUserId = useCurrentUserId();
+    const currentUserId = useCurrentUserIdFromClient();
```

Comment:
@marcbouchenoire @GuillaumeSalles I've not changed this one, but notice how my pure refactoring here shows that this was/is always calling the useCurrentUserId from the Liveblocks context here. Even if you would have a <RoomProvider> in there. I think that's potentially confusing, right? I think we'll want to simply use useCurrentUserId() here, which will call the appropriate method on either the client or the room, depending on the context.

Reply:
I don't know if it's confusing since their difference is an implementation detail and they return the same information

Reply:
I would like to and can clean this up even later so we don't even have to have those two different implementations at all, but that's a post-2.0 thing, an 100% an implementation detail. So no rush with that.

Reply:
OK, I just made the decision to do it: 551f5d460b46d93c2618830e298b299a9b7954df.

@GuillaumeSalles Lmk if I'm missing something here, as you were the original author for this part.